SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -6751,7 +6829,7 @@ void CIncEulerSolver::LoadRestart(CGeometry **geometry, CSolver ***solver, CConf
   }
   
   /*--- Update the old geometry (coordinates n and n-1) in dual time-stepping strategy ---*/
-  if (dual_time && grid_movement)
+  if (dual_time && grid_movement && (config->GetKind_GridMovement() != RIGID_MOTION))
```

Comment:
This just shows what we were discussing a couple of days ago, @talbring, that we should somehow distinguish between rigid grid motion and deformable grids. In rigid grid motion the Grid Velocity is an independent variable of the problem, while in deformable meshes is just derived from Solution_time_n and Solution_time_n1 in second order.